src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1609,11 +1613,7 @@ private bool CompileAndExecute(string code)
 
         private bool CompileAndExecute(List<AssociativeNode> astList)
         {
-            bool succeeded = false;
-            if (astList.Any())
-            {
-                succeeded = Compile(astList, runnerCore);
-            }
+            var succeeded = Compile(astList, runnerCore);
```

Comment:
Ok, it looks like you moved this check inside `Compile`. In that case, the above changes should be fine.